### ISSUE TITLE
Move Game methods to GameManager

### ISSUE THREAD

**Issue by Nolankicks** [Facepunch/sbox-issues#5049]
### What it is?

I think having two very similar methods that can be confusing to new coders that could be easily me merged together. 

### What should it be?

I think one of the methods should be merged into each other. So the methods from Game get merged into GameManager or vice versa.

**Comment by CarsonKompon** [comment]
I can see this. Would be down for GameManager to be merged into Game since I always have to reference the full namespace whenever I make my own "GameManager" component (since it conflicts with Sandbox's)

**Comment by garrynewman** [comment]
Yep I agree with this. GameManager existed really to manage the current scene when the scene system was an addon, makes sense to de-clutter.

**Comment by DevulTj** [comment]
GameManager is obsolete, all related methods and properties have moved to Game
